tests/test_FIA_MS_tools.py

```diff
@@ -0,0 +1,175 @@
+import unittest
+import pandas as pd
+import pickle
+import sys
+import pathlib
+
+sys.path.insert(1, "../")
+from BFAIR.FIA_MS.FIA_MS_tools import (  # noqa E402
+    extractNamesAndIntensities,
+    calculateMeanVarRSD,
+)
+
+current_dir = str(pathlib.Path(__file__).parent.absolute())
+
+
+class test_methods(unittest.TestCase):
+    def setUp(self):
+        file_obj = open(
+            current_dir + "/test_data/FIA_MS_Data/unittest_data/test_data.obj",
+            "rb",
+        )
+        (
+            intensities_triplicates,
+            stats_triplicates,
+            intensities_single,
+            stats_single,
+            intensities_standard,
+            stats_standard,
+        ) = pickle.load(file_obj)
+        file_obj.close()
+
+        self.intensities_triplicates = intensities_triplicates
+        self.stats_triplicates = stats_triplicates
+        self.intensities_single = intensities_single
+        self.stats_single = stats_single
+        self.intensities_standard = intensities_standard
+        self.stats_standard = stats_standard
+
+    def test_extractNamesAndIntensities_triplicates(self):
+        intensities_triplicates = self.intensities_triplicates
+        feature_dir = (
+            current_dir + "/test_data/FIA_MS_Data/features_AdditionalAdducts"
+        )
+        sequence_triplicates = pd.read_csv(
+            current_dir + "/test_data/FIA_MS_Data/sequence_EColi.csv", sep=";"
+        )
+        sample_names_triplicates = sequence_triplicates[
+            "sample_group_name"
+        ].unique()
+        database_triplicates = pd.read_csv(
+            current_dir
+            + "/test_data/FIA_MS_Data/CHEMISTRY/iJO1366_struct.tsv",
+            sep="\t",
+            header=None,
+        )
+        intensities_triplicates_ = extractNamesAndIntensities(
+            feature_dir, sample_names_triplicates, database_triplicates
+        )
+        self.assertEqual(
```

Comment:
Could you also add in a check for the values in the dataframe?  Same for the tests further down.